tsconfig.json

```diff
@@ -1,12 +1,13 @@
 {
   "compilerOptions": {
+    "importHelpers": true,
```

Comment:
importHelpers makes typescript require __extends helper from tslib instead of inlining it to every class declaration